docs/how-to-install-upgrade-downgrade-kubectl-helm.md

```diff
@@ -0,0 +1,138 @@
+This page shows you how to install, upgrade and downgrade kubectl and helm on macOS, Linux (Ubuntu) and Windows via the packages:
+
+- Homebrew on macOS
+- Snap on Linux (Ubuntu)
+- Chocolatey on Windows
+
+# Requirements:
+
+- homebrew available on macOS
+- chocolatey available on Windows
+- Ubuntu is preferred Linux distro
+
+
+# Installing, upgrading and downgrading kubectl on your computer
+``kubectl`` is the Kubernetes command line tool, which can be used to deploy settings to the cluster.
+In this section, you should know how to install, upgrade and downgrade a version of kubectl. 
+
+## Installing kubectl
+
+With the guide below, the latest version of kubeclt should be installed.
+
+
+
+- On macOS, follow the official documentation at https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos.
+
+- On Linux (Ubuntu), follow the official documentation at https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-snap-on-ubuntu.
+
+- On Windows, follow the official documentation at https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-chocolatey-on-windows.
+
+Note: On Windows, to install kubectl with the autocompletion, you should add the option `-y` at the end of the command.
+
+```
+choco install kubernetes-cli -y
+```
+
+## Upgrading kubectl
+This section shows you how to upgrade the latest verion of kubeclt.
+
+- On macOs:
+
+```
+brew upgrade kubernetes-cli
+```
+
+- On Linux (Ubuntu):
+
+//TODO
+
+- On Window, use the command:
+
+```
+choco upgrade kubernetes-cli -y
+```
+
+
+## Downgrading kubectl 
+
+This section shows you how to install an older version of kubectl, e.g `verion 1.11`, use the command below:
+
+- On macOs: See the details at https://www.benpickles.com/articles/72-downgrading-kubectl-with-homebrew
+
+- On Linux (Ubuntu):
+
+//TODO

```

Comment:
I can't find document about binary to install kubectl on linux 32 bit. So this is only for linux 64 bit only. This script is both for upgrade and install
```
$ sudo rm /usr/local/bin/kubectl
$ curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.11.0/bin/linux/amd64/kubectl
$ chmod +x ./kubectl
$ sudo mv ./kubectl /usr/local/bin/kubectl
```